.gitignore

```diff
@@ -58,6 +60,9 @@ _UpgradeReport_Files/
 *.wixobj
 /tools/msvs/genfiles/
 /npm.wxs
+/tools/msvs/msi/Release/
+/tools/msvs/msi/obj/
+/tools/msvs/msi/x64/
```

Comment:
Is there an equivalent `/tools/msvs/msi/x86/`?

Reply:
It's the `Release/` folder. By default, VS uses `Release/` for x86 and `x64/Release/` for x64.